Backend/lib/shared-services/src/IssueManagement/Domain/Action/UI/Issue.hs

```diff
@@ -908,10 +908,6 @@ getConfigValue language issueConfig mbRideInfoRes key = do
         "ESTIMATED_FARE" -> show estimatedFare
         "FINAL_FARE" -> show finalFare
         "FARE_DIFFERENCE" -> show fareDifference
```

Comment:
why did you remove this?

Reply:
oh sorry i accidently committed this causes issue in running dsl because of using rupee sign.